starter-code/jquery_ajax/main.js

```diff
@@ -1,18 +1,23 @@
-/*
 
-- Sign up for openweathermap.org and generate an API key.
-- User either $.ajax or $.get to pull weather current data .
-  for Washington DC (hint: http://api.openweathermap.org/data/2.5/weather?q=...).
-- Print the temperature in console.
-- Bonus 1: add a form prompting user for the city and state.
-- Bonus 2: convert answer from kelvin to fahrenheit.
+'use strict';
+(function request() {
+  var HttpRequest = new XMLHttpRequest();
 
-*/
+   HttpRequest.onreadystatechange = responseM;
+   function responseM(){
 
-'use strict';
-(function() {
-  var weatherUrl = "http://api.openweathermap.org/data/2.5/weather?q=";
-  var apiKey = "";
+	   if (HttpRequest.readyState === XMLHttpRequest.DONE) {
+	     if (HttpRequest.status === 200) {
+	       console.log(HttpRequest.response);
+	     } else {
+	       console.log('request error');
+	     }
+
+	  }
+    document.getElementById('city').addEventListener("click", request);
+	  HttpRequest.open('GET', ' http://api.openweathermap.org/data/2.5/weather');
```

Comment:
Here you need to pass the city name and API key
```suggestion
	  HttpRequest.open('GET', ' http://api.openweathermap.org/data/2.5/weather?q=CITY_NAME&appid=API_KEY');
```